main/test/flix/Test.Exp.Jvm.GetField2.flix

```diff
@@ -0,0 +1,9 @@
+mod Test.Exp.Jvm.InvokeMethod2 {
+
+    import java.awt.Point
+
+//    @test
```

Comment:
I suppose this will be enabled later?

Reply:
I just put it there to start building the house, but I can also just delete it